sandy/njoy.py

```diff
@@ -593,8 +593,8 @@ def _acer_input(endfin, pendfin, aceout, dirout, mat,
 
 def _errorr_input(endfin, pendfin, gendfin, errorrout, mat,
                   ign=2, ek=None,
-                  iwt=2,
-                  temp=NJOY_TEMPERATURES[0],
+                  iwt=2, iread=0,
```

Comment:
Could you also add and test a kwarg `relative=True`, to replace the irelcov keyword of njoy